packages/pyodide-kernel/src/kernel.ts

```diff
@@ -24,7 +24,7 @@ export class PyodideKernel extends BaseKernel implements IKernel {
     super(options);
     this._worker = this.initWorker(options);
     this._worker.onmessage = (e) => this._processWorkerMessage(e.data);
-    this._remoteKernel = wrap(this._worker);
+    this._remoteKernel = coincident(this._worker) as IPyodideWorkerKernel;
```

Comment:
`coincident` takes a second options argument, haven't dug into what they do. 

Perhaps this could be moved into an `protected _initRemote(options): IPyodideWorkerKernel` which can be overloaded by a downstream.